packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -152,6 +156,27 @@ export const findStrategy = function (selector: SelectorStrategy, isW3C?: boolea
         using = 'shadow'
         value = stringSelector.slice(DEEP_SELECTOR.length)
         break
+    // .//*[@aria-labelledby = string(.//*[normalize-space() = "target 2"]/@id)]
+    case 'aria': {
+        const label = stringSelector.slice(ARIA_SELECTOR.length)
+        const conditions = [
+            // element has direct aria label
+            `.//*[@aria-label = "${label}"]`,
+            // aria label is recevied by other element with aria-labelledBy
+            `.//*[@aria-labelledby=(//*[normalize-space() = "${label}"]/@id)]`,
+            // aria label is received from element content
+            `.//*[normalize-space() = "${label}"]`,
```

Comment:
This would retrieve any element containing this text in any of it's child elements.
I believe that instead we need `//*[text() = "${label}"]`

Reply:
I can not really see a difference between `.//` and `//` given that we scan the whole document anyway. See http://xpather.com/1iZkrkxL